src/app/components/route-block/index.js

```diff
@@ -94,6 +94,21 @@ export function RouteBlock() {
 										) ) }
 									</ul>
 								</div>
+								<div>
+									<div>
+										<h2>Supports</h2>
+									</div>
+									<ul className="bb-route-block__supports">
+										{ Object.entries(
+											blockType.supports
+										).map( ( [ key, support ] ) => (
+											<li key={ key }>
+												<strong>{ key }</strong>{ ' ' }
+												{ `- ${ support.toString() }` }
```

Comment:
Do you think JSON.stringify is better than .toString? I'm thinking of situations where the support value is an object?

Reply:
Hey cool! Never thought of that! Fixed  👍